ID/src/main/kotlin/helper/FileHelper.kt

```diff
@@ -3,11 +3,37 @@ package helper
 import java.io.BufferedWriter
 import java.io.File
 import java.io.FileWriter
+import java.io.FileReader
+import java.io.BufferedReader
+import model.Options
+import screen.UserScreen
+import java.lang.StringBuilder
+
 
 object FileHelper {
 
     private const val HEADER = "Name,Age,Gender,Height,Address,Contact"
 
+    fun checkOptionsWhenStart(file: File){
```

Comment:
I think this method should in other Class. Something like main screen.
It is option to interact with user. So it will identify as screen class.